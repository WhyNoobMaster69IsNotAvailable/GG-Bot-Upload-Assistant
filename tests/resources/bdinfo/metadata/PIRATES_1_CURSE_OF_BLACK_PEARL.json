{
    "raw_video_file": "/data/downloads/FullDisk/PIRATES_1_CURSE_OF_BLACK_PEARL/BDMV/STREAM/00294.m2ts",
    "raw_file_name": "PIRATES_1_CURSE_OF_BLACK_PEARL",
    "mediainfo":" /app/temp_upload/63d54771af6760c164221154515df928e0e62749242fe71400539fa761099cc2/mediainfo.txt",
    "largest_playlist": "00800.MPLS",
    "upload_media": "/data/downloads/FullDisk/PIRATES_1_CURSE_OF_BLACK_PEARL/",
    "bdinfo_output_split": "b'Please wait while we scan the disc...\n# Group Playlist File Length Estimated Bytes Measured Bytes \n\n1 1 00800.MPLS 02:23:09 63,866,038,272 - \n2 1 00801.MPLS 02:23:09 63,866,038,272 - \n3 1 01309.MPLS 02:23:09 63,866,038,272 - \n4 2 00156.MPLS 00:01:00 327,942,144 - \n5 3 00157.MPLS 00:01:00 328,574,976 - \n6 4 01621.MPLS 00:00:36 7,176,192 - \n7 4 01625.MPLS 00:00:35 7,151,616 - \n8 5 01478.MPLS 00:00:21 11,563,008 - \n9 5 01133.MPLS 00:00:21 11,538,432 - \n10 6 01476.MPLS 00:00:20 46,546,944 - \n(*) Some playlists on this disc have hidden tracks. These tracks are marked with an asterisk.\n'"
}
