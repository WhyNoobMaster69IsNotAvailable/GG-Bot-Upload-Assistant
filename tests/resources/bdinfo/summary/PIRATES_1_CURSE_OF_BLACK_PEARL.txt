
QUICK SUMMARY:

Disc Title: Pirates of the Caribbean: The Curse of the Black Pearl - Ultra HDâ„¢
Disc Label: PIRATES_1_CURSE_OF_BLACK_PEARL
Disc Size: 64,987,547,038 bytes
Protection: AACS2
Playlist: 00800.MPLS
Size: 63,866,038,272 bytes
Length: 2:23:09.247
Total Bitrate: 59.48 Mbps
Video: MPEG-H HEVC Video / 45389 kbps / 2160p / 23.976 fps / 16:9 / Main 10 @ Level 5.1 @ High / 10 bits / HDR10 / BT.2020
Audio: English / Dolby TrueHD/Atmos Audio / 7.1 / 48 kHz /  4115 kbps / 24-bit (AC3 Embedded: 5.1-EX / 48 kHz /   640 kbps / DN -27dB)
Audio: English / Dolby Digital Audio / 2.0 / 48 kHz /   320 kbps / DN -24dB
Audio: French / Dolby Digital Plus Audio / 7.1 / 48 kHz /  1024 kbps / DN -27dB (AC3 Embedded: 5.1 / 48 kHz /   576 kbps / DN -27dB)
Audio: French / Dolby Digital EX Audio / 5.1-EX / 48 kHz /   640 kbps / DN -27dB
Audio: Spanish / Dolby Digital Plus Audio / 7.1 / 48 kHz /  1024 kbps / DN -27dB (AC3 Embedded: 5.1 / 48 kHz /   576 kbps / DN -27dB)
Audio: German / Dolby Digital Plus Audio / 7.1 / 48 kHz /  1024 kbps / DN -27dB (AC3 Embedded: 5.1 / 48 kHz /   576 kbps / DN -27dB)
Audio: Japanese / Dolby Digital Plus Audio / 7.1 / 48 kHz /  1024 kbps / DN -27dB (AC3 Embedded: 5.1 / 48 kHz /   576 kbps / DN -27dB)
Subtitle: English / 32.670 kbps
Subtitle: French / 25.789 kbps
Subtitle: French / 31.225 kbps
Subtitle: Spanish / 29.751 kbps
Subtitle: German / 27.476 kbps
Subtitle: Japanese / 21.192 kbps
Subtitle: Dutch / 22.944 kbps
Subtitle: Danish / 27.902 kbps
Subtitle: Finnish / 29.656 kbps
Subtitle: Norwegian / 24.080 kbps
Subtitle: Swedish / 25.228 kbps
Subtitle: French / 0.060 kbps
Subtitle: French / 0.064 kbps
Subtitle: Spanish / 0.099 kbps
Subtitle: German / 0.038 kbps
Subtitle: Japanese / 0.042 kbps
