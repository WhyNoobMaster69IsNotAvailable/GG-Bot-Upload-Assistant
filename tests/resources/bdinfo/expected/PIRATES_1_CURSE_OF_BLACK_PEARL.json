{
    "video": [
        {
            "codec": "HEVC",
            "bitrate": "45389 kbps",
            "resolution": "2160p",
            "fps": "23.976 fps",
            "aspect_ratio": "16:9",
            "profile": "Main 10 @ Level 5.1 @ High",
            "bit_depth": "10 bits",
            "dv_hdr": "HDR10",
            "color": "BT.2020"
        }
    ],
    "audio": [
        {
            "language": "English",
            "atmos": "Atmos Audio",
            "codec": "Dolby TrueHD",
            "channels": "7.1",
            "sample_rate": "48 kHz",
            "bitrate": "4115 kbps",
            "bit_depth": "24-bit"
        },
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "320 kbps",
            "bit_depth": "DN -24dB"
        },
        {
            "language": "French",
            "codec": "Dolby Digital Plus Audio",
            "channels": "7.1",
            "sample_rate": "48 kHz",
            "bitrate": "1024 kbps",
            "bit_depth": "DN -27dB"
        },
        {
            "language": "French",
            "codec": "Dolby Digital EX Audio",
            "channels": "5.1-EX",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -27dB"
        },
        {
            "language": "Spanish",
            "codec": "Dolby Digital Plus Audio",
            "channels": "7.1",
            "sample_rate": "48 kHz",
            "bitrate": "1024 kbps",
            "bit_depth": "DN -27dB"
        },
        {
            "language": "German",
            "codec": "Dolby Digital Plus Audio",
            "channels": "7.1",
            "sample_rate": "48 kHz",
            "bitrate": "1024 kbps",
            "bit_depth": "DN -27dB"
        },
        {
            "language": "Japanese",
            "codec": "Dolby Digital Plus Audio",
            "channels": "7.1",
            "sample_rate": "48 kHz",
            "bitrate": "1024 kbps",
            "bit_depth": "DN -27dB"
        }
    ],
    "title": "Pirates of the Caribbean: The Curse of the Black Pearl - Ultra HD\u2122",
    "label": "PIRATES_1_CURSE_OF_BLACK_PEARL",
    "size": 60.524369625374675,
    "playlist": "00800.MPLS",
    "length": "2:23:09"
}